format.go

```diff
@@ -16,7 +13,7 @@ const (
 type Units uint
 
 const (
-	UnitNone Units = iota
```

Comment:
I'll review your PR tomorrow, meanwhile can you please explain what is the benefit of making test in different package like "mpb_test"? Why not use just same package "mpb"?